dockerfiles/Dockerfile.heroku-24

```diff
@@ -0,0 +1,17 @@
+FROM heroku/heroku:24-build
+
+USER root
+
+# setup workspace
+RUN rm -rf /tmp/workspace
+RUN mkdir -p /tmp/workspace
+
+RUN apt-get update -y; apt-get install ruby -y
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
Why is rust installed here?

Reply:
Rust is needed to support the `--yjit` flag https://shopify.engineering/ruby-yjit-is-production-ready